typing/env.ml

```diff
@@ -281,29 +492,35 @@ let is_in_signature env = env.flags land in_signature_flag <> 0
 let is_implicit_coercion env = env.flags land implicit_coercion_flag <> 0
 
 let diff_keys is_local tbl1 tbl2 =
```

Comment:
`diff_keys(2)` could be lifted inside the `EnvTbl(2)` module.
Also, the only purpose of `EnvTbl.local_keys` is to implement `diff_keys`.

So all could be replaced by just an export of `EnvTbl.diff_keys` & `EnvTbl2.diff_keys`.